CHANGELOG.md

```diff
@@ -89,5 +89,7 @@
 - BEIS users can view Transactions & Budgets on a project, but not create or edit them
 - Country list for recipient countries when creating an activity has been reduced to only those ODA uses as recipients.
 - Add feedback form link to phase banner
+- When creating an activity the Finance step has been defaulted to "Standard grant" and omitted from the user journey
+- Finance column has been deleted from activities table on the schema
```

Comment:
I am not sure we need this in the Changelog at all, but it is not in this commit eitherway?

I like to think of the changelog as features a user might understand, I guess there is an exception for security fixes so maybe 'something the product owner understands' is a better description?

Reply:
Let's talk about that later. I might have not understood the purpose of a Changelog.

Reply:
I've amended this file. Now it's showing the right information